server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -608,7 +624,15 @@
   };
 
   function addURLQueryParams(reportType, url){
-    return url + `${reportType.queryParams ? `${url.includes('?') ? '&' : '?'}${new URLSearchParams(_.clone(reportType.queryParams)).toString()}` : ''}`
+    let queryParams = _.clone(reportType.queryParams);
+    if(url.includes('?')) {
+      const urlQueryParams = url.split('?')[1].split('&').map((param) => param.split('=')[0])
+      urlQueryParams.forEach((param) => {
+        delete queryParams[param]
+      })
+    }
```

Comment:
I think here you can use the url module from node, would probably make it easier to parse any existing url params.

```js
const _url = require('url');
const parsedUrl = _url.parse(url);
if (parsedUrl && parsedUrl.query) {
  // do your code
}
```

Reply:
updated.

Reply:
actually I read on the url.parse function and it is deprecated. the recommended does not really work here without the base domain so I reverted to your original changes 😄